marklogic-client-api/src/test/java/com/marklogic/client/test/junit5/RequireSSLExtension.java

```diff
@@ -52,6 +59,39 @@ public void afterAll(ExtensionContext context) {
 		new CertificateTemplateManager(manageClient).delete(TEMPLATE);
 	}
 
+	/**
+	 * @return a trust manager that accepts the public certificate associated with the certificate template created
+	 * by this class.
+	 */
+	public static X509TrustManager newTrustManager() {
+		return new X509TrustManager() {
+			@Override
+			public void checkClientTrusted(X509Certificate[] chain, String authType) {
+			}
+
+			@Override
+			public void checkServerTrusted(X509Certificate[] chain, String authType) {
+			}
+
+			@Override
+			public X509Certificate[] getAcceptedIssuers() {
+				return new X509Certificate[]{getCertificate()};
+			}
+		};
+	}
+
+	private static X509Certificate getCertificate() {
+		CertificateTemplateManager mgr = new CertificateTemplateManager(Common.newManageClient());
+
+		Fragment response = mgr.getCertificatesForTemplate(TEMPLATE_NAME);
+		String cert = response.getElementValue("/msec:certificate-list/msec:certificate/msec:pem");
```

Comment:
@BillFarber Could reproduce this in mlxprs if needed, though I don't know how to make a certificate / trust manager in Node or what the equivalent is. But this is the key part - after generating a temporary certificate, you can download it and get the pem via the above code.

Reply:
I've been thinking that we need to add a test or two to exercise certs, but I haven't written any stories for it.